README.md

```diff
@@ -41,6 +41,42 @@ Fedora Remix for WSL is provided on a community-support basis. There is no suppo
 
 Whitewater Foundry devs monitor the Fedora Remix for WSL GitHub issues page; however, users who require dedicated end-user support for WSL should consider [Pengwin](https://github.com/WhitewaterFoundry/Pengwin) instead. Businesses and other organizations that would like to receive ongoing professional support should [e-mail us](mailto:enterprise@whitewaterfoundry.com) or visit [our website](https://www.whitewaterfoundry.com/wlinux-enterprise-edition/) to learn more about Pengwin Enterprise.
 
+### In-place Upgrading
+You can upgrade your existing installation of Fedora Remix with the following steps. 
+
+First, backup your installation:
+
+`wsl --export fedoraremix fedoraremix_backup.tar.gz`
+
+Then proceed:
+
+> NOTE: DO NOT PERFORM THE UPGRADE STARTING FEDORA REMIX WITH SYSTEMD; IT WILL RUIN YOUR INSTALLATION
+
+#Be sure that systemd is disabled
+`[ "$(grep -c "^systemd.*=.*true$" /etc/wsl.conf)" -ne 0 ] && sudo sed -i "s/^systemd.*=.*true$/systemd=false/" /etc/wsl.conf && wsl.exe --terminate ${WSL_DISTRO_NAME}`
+
+```bash
+next_version=$(echo "${VERSION_ID}+1" | bc)
```

Comment:
What is the source of VERSION_ID? This is not set for me (but I am on Fedora 38 already)

I also understand the intent on doing the +1, as this matches the Fedora recommendation to always upgrade through versions, but I am also torn between this here and making the user responsible for explicitly entering the correct version. Personally, I'd prefer making the user responsible.

Reply:
I forgot to add this line

```bash
source /etc/os-release
```

This defines VERSION_ID
But you are right I'll change it to 
```bash
next_version=38 #Put the version you want to upgrade to here
```